res/CircleSchema/0.8/circle_schema.fbs

```diff
@@ -442,52 +442,6 @@ enum BuiltinOperator : int32 {
   BITCAST = 159,
   BITWISE_XOR = 160,
   RIGHT_SHIFT = 161,
-  // All Operators start with STABLEHLO_ prefixes are subject to change
```

Comment:
you shouldn't update this file for onert-micro

Reply:
oops, it is mistake...